lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -56,7 +56,7 @@ def define
 
       #Q4
       checkbox_question :receiving_non_exemption_benefits? do
-        benefits.each do |benefit|
+        benefits.keys.each do |benefit|
```

Comment:
Can we guarantee that these `benefits.keys` will always come back in the same order? Am I right in thinking that if the order changes then the order of the benefit options will change for the user?